geode-management/src/main/java/org/apache/geode/management/api/CliResponse.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.api;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CliResponse {
+  private String httpMethod;
+  private String url;
+
+  public CliResponse() {
+  }
+
+  public CliResponse(String httpMethod, String url) {
```

Comment:
This class can be made immutable by removing the setter methods and empty constructor, making the member fields final, and adding json annotation to this constructor.  See lines 38-40 of ClusterManagementListOperationsResult for an example.